docs/architecture_and_project_structure.md

```diff
@@ -4,13 +4,13 @@
 
 ### Presentation
 
-Logic in a presentation layer can be either business logic(Model) or UI logic(WidgetModel):
-* Business logic is the implementation of product requirements for application data. For example, bookmarking an article in a news reader app when a user clicks on a button.
-* UI logic is related to how to display UI state on the screen. For example, obtaining the right search bar hint when the user has selected a category, scrolling to a particular item in a list, or the navigation logic to a particular screen when the user clicks a button.
+Logic in a presentation layer can be either **Business logic**(Elementary Model) or **UI logic**(Widget Model):
+* **Business logic** is the implementation of product requirements for application data. For example, adding an item to the cart when the user clicks on a button.
+* **UI logic** is related to how to display UI state on the screen. For example, displaying or hiding the load indicator depending on the data load status or showing a snack bar.
```

Comment:
displaying or hiding load indicator is not a UI logic here. You can leave the example with snack bar and navigation.